static/index.html

```diff
@@ -268,6 +272,10 @@
             </a>
           </div>
         </div>
+        <div v-if="postsPerPage > 0 && postLocations !== null  && postLocations.length > postsPerPage" class="center pagination flex-center">
```

Comment:
Just me being picky ahah

```suggestion
        <div v-if="postsPerPage > 0 && postLocations !== null && postLocations.length > postsPerPage" class="center pagination flex-center">
```